source/borderrouter_thread_tasklet.c

```diff
@@ -147,10 +146,11 @@ static int thread_interface_up(void)
     device_config.PSKd_len = len;
     memset(device_config.PSKd_ptr, 0, len + 1);
     memcpy(device_config.PSKd_ptr, param, len);
+    rf_read_mac_address(rf_mac);
+    memcpy(device_config.eui64, rf_mac, 8);
```

Comment:
Stack reads the eui64 from radio if initialised for 0 is this value not initialised properly?